tests/console/force_cron_run.pm

```diff
@@ -30,18 +30,27 @@ sub settle_load {
 
 sub run {
     select_console 'root-console';
-
     # show dmesg output in console during cron run
     assert_script_run "dmesg -n 7";
 
     # Make sure there's no load before we trigger one via cron.
     settle_load;
     my $before = time;
-    assert_script_run "bash -x /usr/lib/cron/run-crons", 1000;
+    # "bash -x /usr/lib/cron/run-crons" doesn't work really on a running system with uptime less than 15 minutes
+    # because no daily jobs got created or can be excecuted.
```

Comment:
What exactly do you mean with "no daily jobs got created". Do you mean that symlinks in the folders /etc/cron.*/ are not yet there at the time of calling /usr/lib/cron/run-crons?